CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ option(FRENCTOOLS "Enable building fre-nctools.fd" ON)
 option(GRIDTOOLS "Enable building grid_tools.fd" ON)
 option(CHGRES "Enable building chgres_cube.fd" ON)
 option(OROG_MASK_TOOLS "Enable building orog_mask_tools.fd" ON)
+# OROG_MASK_TOOLS must ON for OROG_NETCDF to build.
+option(OROG_NETCDF "Enable building orog_netcdf.fd" OFF)
```

Comment:
this name had me a little confused at first, I thought there were separate directories for the binary and netcdf read options. since there already is a separate "gsl" option, I suggest adding "tools" to the end of this directory: orog_netcdf_tools

Reply:
Done at 06291a4.